content/community/moderation-tools.md

```diff
@@ -0,0 +1,24 @@
+---
+kind: reference
+sidebar: community
+prev: /community/moderation/
+next: /community/rules/
+---
+
+# Moderation Tools
+
+[Moderators](/community/moderation/#moderators) are a group of users selected to support users with handling problems related to content and community. They have access to tools and features not available to regular users, which can be used to solve problems reported by community members. In particular, moderators can:
+
+- Handle problematic users, cheaters, and griefers:
+  - Handle reports about dishonest users,
+  - Flag solutions as cheat,
```

Comment:
I'm not sure about English writing practices, but I would think that what follows a comma is continuation of the sentence and thus should start with a lowercase letter. Although English already has a weird casing in titles, so I don't know what else there can be...

Reply:
When it comes to such things, I count on native English speakers here ;)

Reply:
I'm not really sure either, but I'd probably remove the commas, due to what Unnamed just said...

Reply:
Yeah, remove the commas.

I don't think I've seen this style before. It's usually either a list of short phrases without any trailing punctuation (comma/period) or a list of complete sentence(s) with periods.

Reply:
Removed commas.